web/src/features/cHaines/slices/cHainesModelRunsSlice.tsx

```diff
@@ -0,0 +1,189 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { ModelRun, ModelRuns, getModelRuns, getCHainesGeoJSON } from 'api/cHainesAPI'
+import { AppThunk } from 'app/store'
+import { logError } from 'utils/error'
+import { FeatureCollection } from 'geojson'
+
+interface State {
+  loading: boolean
+  error: string | null
+  model_runs: ModelRun[]
+  model_run_predictions: Record<string, Record<string, Record<string, FeatureCollection>>>
+  model_run_predictions_status: Record<string, Record<string, Record<string, string>>>
+  selected_model_abbreviation: string
+  selected_model_run_timestamp: string
+  selected_prediction_timestamp: string
+}
+
+interface GeoJSONContext {
+  model: string
+  model_run_timestamp: string
+  prediction_timestamp: string
+  result: FeatureCollection
+}
+
+const initialState: State = {
+  loading: false,
+  error: null,
+  model_runs: [],
+  selected_model_abbreviation: 'GDPS',
```

Comment:
why is this set to GDPS by default?

Reply:
Good question - I'll throw it at people in standup.

Reply:
Changed to HRDPS - but there's other code in the UI that overrides it. Have added this into a ticket.